common/components/utils/CurrentUser.js

```diff
@@ -25,6 +27,13 @@ class CurrentUser {
   static isStaff() : boolean {
     return window.DLAB_GLOBAL_CONTEXT.isStaff;
   }
+
+  static isCoOwner(project: ProjectDetailsAPIData) {
```

Comment:
Make sure to document the fact that the project creator does _not_ count as a co-owner.

Reply:
Added comments to currentuser.js and volunteersection.jsx